src/common/dao/user.go

```diff
@@ -234,6 +234,13 @@ func OnBoardUser(u *models.User) error {
 	}
 	if created {
 		u.UserID = int(id)
+		// current orm framework doesn't support insert null value
```

Comment:
Can we open an issue on beego's repo, and reference that issue?

Reply:
Updated